apps/src/turtle/skins.js

```diff
@@ -42,18 +42,29 @@ exports.load = function (assetUrl, id) {
     skin[prop] = config[prop];
   }
 
-  // TODO (br-pair) : Some of these keys are actually undefined. Clean this up
-  skin.lineStylePatternOptions = [
-    [skin.patternDefault, 'DEFAULT'], //  signals return to default path drawing
-    [skin.rainbowMenu, 'rainbowLine'],  // set to property name for image within skin
-    [skin.ropeMenu, 'ropeLine'],  // referenced as skin[pattern];
-    [skin.squigglyMenu, 'squigglyLine'],
-    [skin.swirlyMenu, 'swirlyLine'],
-    [skin.annaLine, 'annaLine'],
-    [skin.elsaLine, 'elsaLine'],
-    [skin.annaLine_2x, 'annaLine_2x'],
-    [skin.elsaLine_2x, 'elsaLine_2x'],
-  ];
+  // Declare available line style patterns. This array of arrays is eventually used
+  // to populate the image dropdown in the Set Pattern block.
+
+  // Start by always including the default pattern
+  skin.lineStylePatternOptions = [[skin.patternDefault, 'DEFAULT']];
+
+  // Then selectively include other line patterns depending on the skin.
+  // Anna and Elsa probably shouldn't draw other line patterns other than their own.
+  // When they try to draw other patterns, it looks very distorted.
+  // This is likely because there was lot hand-tweaking done previously to make those
+  // line patterns look good, and we don't want to touch that.
+
+  // With Anna and Elsa, we're only including their own patterns in the dropdown.
+  if (skin.id == "anna") {
```

Comment:
how about using the CONFIGS array above instead of adding new branching logic here?